### PR TITLE
[DOCS-185] Agile Pentesting, New Pentest Wizard

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
@yogipetkar As promised in DOCS-185, I'm submitting documentation changes for Agile Pentesting. This is the first draft to understand your expectations for documentation and get your review on some important concepts such as Pentest Types and Automated Report.

We still have time to edit content before October 7.

This is how the guides will appear on the portal:

- Pentest Types
https://deploy-preview-168--cobalt-docs.netlify.app/platform-deep-dive/pentests/pentest-types/

- Pentest Reports
https://deploy-preview-168--cobalt-docs.netlify.app/platform-deep-dive/pentests/reports/

- Updated Let's Get Started screen
https://deploy-preview-168--cobalt-docs.netlify.app/getting-started/assets/

### CODE REVIEW

File: content/en/Platform Deep Dive/Pentests/Reports/_index.md [hunk 1]
```diff
@@ -10,17 +10,19 @@ description: >
...
-| Full Report + Finding Details | Adds details of every test finding to the full report. Details include:<br>  <br>- Vulnerability Type <br>- Description <br>- Proof of Concept <br>- Severity <br>- Suggested Fix     | 
-| Customized Report             | After you [customize](./customize-report) a report, you'll see this option. You can start from any of the other report templates, except the Attestation Letter. |
+| Report Type | Available for [Pentest Type](/platform-deep-dive/pentests/pentest-types/) | Description | [Customizable](/platform-deep-dive/pentests/reports/customize-report/) |
+|-----|-----|-----|-----|
+| Automated Report | Agile Pentest | A system-generated report for an [Agile Pentest](/getting-started/glossary/#agile-pentest) intended for internal use. Includes the following sections:<br><br><li>Pentester user information</li><li>Executive Summary</li><li>Methodology</li><li>Post-Test Remediation</li><li>Finding Details</li><br>You can download the report once we move the pentest to the _Remediation_ [state](/penteststates/). | Yes |
```
Comment (yogipetkar) [inline content/en/Platform Deep Dive/Pentests/Reports/_index.md:19]: @ana-dashuk-cobalt - This ticket captures all sections of the automated report. Even though the once that are identified in the doc are the main ones, please take a look if we need to add more details here.
Additionally, "Pentester user information" => "Pentest information" might read better.

File: layouts/shortcodes/comprehensive-big-scope.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Comprehensive Pentests requiring more than 20 credits don’t get immediate confirmation. We’ll specify the number of required credits after reviewing the pentest.
```
Comment (yogipetkar) [inline layouts/shortcodes/comprehensive-big-scope.md:1]: "...don't get immediate credit confirmation."

File: layouts/shortcodes/agile-pentest.md [hunk 3]
```diff
@@ -0,0 +1 @@
+An Agile Pentest focuses on code changes or a specific area of an asset and comes with an Automated Report intended for internal use
```
Comment (yogipetkar) [inline layouts/shortcodes/agile-pentest.md:1]: There are some additional use cases if you want to capture here.
https://docs.google.com/presentation/d/1Rujaf_XSwW5VdiSHrYuecNjAcrmobx97jYI5SM9NLrg/edit#slide=id.g1431c57e796_4_1

File: content/en/Platform Deep Dive/Pentests/pentest-types.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+| **Minimum Number of Credits per Pentest** | 3 credits | 5 credits<br><li>{{% comprehensive-big-scope %}}</li> |
+| **[Pentest Lead](/getting-started/glossary/#pentest-lead) Assigned** | No | Yes |
+| **Available [Pentest Reports](/platform-deep-dive/pentests/reports/)** | <li>Automated Report</li> | <li>Customer Letter</li><li>Attestation Letter</li><li>Attestation Report</li><li>Full Report</li><li>Full Report + Finding Details</li> |
+| **Report Target Audience** | Internal stakeholders | External stakeholders |
+
+You can change the type of your pentest before we move it to the [Planned](/penteststates/) state. On the pentest brief, navigate to the **Settings** tab, and select the type under **Pentest Type**.
```
Comment (yogipetkar) [inline content/en/Platform Deep Dive/Pentests/pentest-types.md:27]: Either through PT Settings or by clicking Edit in PT Brief view.

**Review by yogipetkar — APPROVED** [review]
With some minor comments, I approve this PR. Trust that the comments will be reviewed and, if appropriate, incorporated.

File: layouts/shortcodes/agile-pentest.md (hunk 3 above)
Comment (mjang-cobalt) [reply layouts/shortcodes/agile-pentest.md:1]: I've set up DOCS-301. Great idea, I love including use cases, but I want us to be able to Merge without delay on Monday, as scheduled.

**Review by mjang-cobalt — APPROVED** [review]
Excellent. Thanks so much for your hard work on this, @ana-dashuk-cobalt

File: layouts/shortcodes/agile-pentest.md (hunk 3 above)
Comment (ana-dashuk-cobalt) [reply layouts/shortcodes/agile-pentest.md:1]: @mjang-cobalt The use cases listed in the presentation are captured in `pentest-types.md`.

Comment (mjang-cobalt) [reply layouts/shortcodes/agile-pentest.md:1]: Quite right, thanks.

Screen Shot 2022-10-03 at 6 16 58 AM

**Pull request merged**